jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/jsonb/JsonbDiagnosticsCollectorTest.java

```diff
@@ -83,18 +89,47 @@ public void JsonbTransientNotMutuallyExclusive() throws Exception {
         JakartaDiagnosticsParams diagnosticsParams = new JakartaDiagnosticsParams();
         diagnosticsParams.setUris(Arrays.asList(uri));
 
-        Diagnostic d1 = d(22, 19, 23,
+        // Diagnostic for the field "name"
```

Comment:
Consider adding these comments to the test class itself, ie:
```java

@JsonbProperty("name")
@JsonbTransient
private String name; // JsonbTransient is mutually exclusive with other JsonB annotations

```

Reply:
Done